website/docs/key_generation.md

```diff
@@ -7,9 +7,13 @@ last_modified_at:
 Every record in Hudi is uniquely identified by a primary key, which is a pair of record key and partition path where the record belongs to.
 Using primary keys, Hudi can impose a) partition level uniqueness integrity constraint b) enable fast updates and deletes on records. 
 One should choose the partitioning scheme wisely as it could be a determining factor for your ingestion and query latency.
+Some use cases do not have a naturally present record key, for ex. log ingestion type of payloads. For these type of use cases,
```

Comment:
@nsivabalan  please review and suggest edits on this part.

Reply:
looks good. 
minor suggestion 
```
compute, storage and read to meet the uniqueness requirements
```

"and" -> "to"

Reply:
also, can we add somewhere that auto record key generation is from 0.14.0 onwards.